src/gfx/pal_packing.cpp

```diff
@@ -3,12 +3,16 @@
 #include "gfx/pal_packing.hpp"
 
 #include <algorithm>
+#include <cstdint>
```

Comment:
How do the `<c*>` headers keep getting added when we have `<*.h>` already :P
```suggestion
```

Reply:
`clangd` only considers the former instead of the latter, for some reason. :pensive: